test/performance/shmem_perf_suite/common.h

```diff
@@ -14,6 +14,21 @@
 #include <string.h>
 #include <assert.h>
 #include <stdbool.h>
+#include <sys/time.h>
+
+#ifndef HAVE_SHMEMX_WTIME
+double
+shmemx_wtime(void)
+{
+    double wtime;
+    struct timeval tv;
+
+    gettimeofday(&tv, NULL);
+    wtime = tv.tv_sec;
+    wtime += (double)tv.tv_usec / 1000000.0;
+    return wtime;
+}
+#endif
```

Comment:
I don't know what shmemx_wtime() is supposed to return. Seconds? This will lose the timer resolution. Should we be reporting ns instead?